examples/tof-viewer/src/ADIToFRecorder.cpp

```diff
@@ -129,22 +129,49 @@ void ADIToFRecorder::stopRecording() {
 }
 
 int ADIToFRecorder::startPlaybackRaw(const std::string &fileName, int &fps) {
+    aditof::Metadata m_metadataStruct;
```

Comment:
Since this variable in not a class member but a local variable, it doesn't need the "m_" prefix.